libs/ledger-live-common/src/apps/logic.test.ts

```diff
@@ -314,7 +314,7 @@ const scenarios = [
           },
         ],
         expectPlan:
-          "-XRP, -Ethereum Classic, -Ethereum, -Dogecoin, -Zcash, -Bitcoin",
```

Comment:
There are many other mentions of zcash in this file, is this the only place that needs to be updated?

Reply:
yes, because this line of code check that if bitcoin and zcash have installed in nano, when we uninstall bitcoin, the zcash app should be uninstalled too. but it is not the case. So I remove zcash here
No impact to the other test cases

Reply:
shouldn't this change made in another PR? i feel we fix 2 topic in same PR. could have been more atomic. because i don't see this topic mentioned in initial ticket.
also wonder about the fact that some old nano app may still need ZCash>Bitcoin dep? this would be covered by a ticket to document the risk if there were a ticket on this topic 🙏

Reply:
I agree, in the future @hzheng-ledger please try to do separate PRs for separate issues, so they can be tested separately and impact is better controlled.

I think for this PR it can be enough to complete the description, but please describe the dependencies and impact: this PR must not be merged before the ZCash Nano app is released, and even then, we need to make sure that the transition is smooth for users with the current (old) zcash app.
I guess it will just be about setting a minimum version for zcash here: https://github.com/LedgerHQ/ledger-live/blob/538ce3a24b72d7fe8ee838e0e32fde3bf38e3ae6/libs/ledger-live-common/src/apps/support.ts#L31